express/code/blocks/simplified-pricing-cards/simplified-pricing-cards.css

```diff
@@ -48,10 +48,6 @@ main .section>.simplified-pricing-cards-wrapper {
 
 }
 
-.simplified-pricing-cards .card:first-of-type {
```

Comment:
I think this causes regression.

See: https://uk-campaign-pricing-cards-outline--express-milo--adobecom.hlx.live/express/business
vs 
https://main--express-milo--adobecom.hlx.live/express/business

"We need a new iteration of the pricing card for the free plan"